packages/profile-sync-controller/src/controllers/user-storage/UserStorageController.test.ts

```diff
@@ -1104,7 +1104,7 @@ describe('user-storage/user-storage-controller - saveInternalAccountToUserStorag
     });
 
     await controller.saveInternalAccountToUserStorage(
-      MOCK_INTERNAL_ACCOUNTS.ONE[0].address,
+      MOCK_INTERNAL_ACCOUNTS.ONE[0] as InternalAccount,
```

Comment:
Just want to make sure, is this assertion valid? If so then all good!

Reply:
Yes definitely! It's just that my data doesn't 100% intersect with type `InternalAccount` since it misses some keys we don't really care about, and I didn't want to bloat up the fixtures.